javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/config/AgentConfigBuilder.java

```diff
@@ -12,12 +12,12 @@
 import java.util.Map;
 import java.util.Properties;
 
-public final class ConfigBuilder
-    extends io.opentelemetry.sdk.common.export.ConfigBuilder<ConfigBuilder> {
+public final class AgentConfigBuilder
+    extends io.opentelemetry.sdk.common.export.ConfigBuilder<AgentConfigBuilder> {
```

Comment:
Really minor: you can now import `ConfigBuilder`

Reply:
good point, let's keep it clean.